util/syspolicy/syspolicy_test.go

```diff
@@ -8,15 +8,22 @@ package syspolicy
 import (
 	"errors"
 	"testing"
+	"time"
 )
 
+// testHandler encompasses all data types returned when testing any of the syspolicy
+// methods that involve getting a policy value.
+// For keys and the corresponding values, check policy_keys.go.
 type testHandler struct {
-	t     *testing.T
-	name  string
-	key   Key
-	value string
-	u64   uint64
-	err   error
+	t                *testing.T
+	name             string
+	key              Key
+	value            string
+	u64              uint64
+	err              error
+	preferenceOption PreferenceOption // used to test GetPreferenceOption
+	visibility       Visibility       // used to test GetVisibility
+	duration         time.Duration    // used to test GetDuration
```

Comment:
I think these would be best moved to a per-test test case struct. Right now it's kind of mixing the mocked Handler functionality with test conditions.